Lib/logging/__init__.py

```diff
@@ -2246,7 +2246,9 @@ def _showwarning(message, category, filename, lineno, file=None, line=None):
         logger = getLogger("py.warnings")
         if not logger.handlers:
             logger.addHandler(NullHandler())
-        logger.warning("%s", s)
+        # Log str(s) directly instead of with logger.warning("%s", s)
+        # since some log aggregation tools group logs by the msg arg
```

Comment:
Please add a reference to the bpo issue number in the comment, so it's easier to get more detail on the reason for the change.

Reply:
Done ✅